spec/rails_helper.rb

```diff
@@ -1,12 +1,14 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
+require 'selenium/webdriver'
+require 'capybara-screenshot/rspec'
 ENV['RAILS_ENV'] ||= 'test'
 require File.expand_path('../../config/environment', __FILE__)
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
-
+include Warden::Test::Helpers
```

Comment:
I think with the `Devise::Test::IntegrationHelpers` module included later, you do not need to include this line too. You can remove this line and your tests should still pass.